docs/config.md

```diff
@@ -318,6 +319,110 @@ location /blog/ {
 
 Default: `[]`
 
+- `NoDNSLink`
+  A boolean to configure whether DNSLink lookup for value in `Host` HTTP header should be performed.
+  If DNSLink is present, content path stored in it is mounted at `/` and respective payload is returned to the client.
+
+  Default: `false`
+
+### Gateway.PublicGateways
+
+`PublicGateways` is a dictionary for overriding default options on specific fully qualified domain names (FQDN).  
+Options for changing behavior per hostname:
+
+- `NoDNSLink`  
+  Overrides global setting of the same name.  
+  Default: `false`
+- `PathPrefixes`  
+  Array of paths that should be mounted at the root of domain name.  
+  **TODO: this does not override global 'PathPrefixes' used for validation of X-Ipfs-Gateway-Prefix header. Should this have different name? eg. PathMounts?**
+  Example: `["/ipfs", "/ipns", "/api"]`  
+  Default: `[]`
```

Comment:
Are we sure we want this to default to empty?

Reply:
I renamed `PathMounts` to `Paths` and removed it from global scope.
Our "know gateways" such as `localhost` are initialized with  `Paths: ["/ipfs", "/ipns"]`:

https://github.com/ipfs/go-ipfs/blob/feat/gateway-subdomains/core/corehttp/hostname.go#L25-L31

For hostnames added by the user, I think  `[]` is a safer default: it requires user to explicitly expose `/ipfs/`, reducing the number of misconfigured DNSLink websites that  act as a public gateways by mistake.